<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTAStrategyManager</class>
 <widget class="QWidget" name="CTAStrategyManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTA策略</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>514</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>添加策略</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>初始化</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>启动</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>停止</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>129</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>全部初始化</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>全部启动</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>全部停止</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>清空日志</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QTableView" name="tableView_2"/>
         <widget class="QTableWidget" name="tableWidget_3">
          <column>
           <property name="text">
            <string>时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>信息</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>接口</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>addStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>startStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>stopStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>initAllStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>startAllStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>stopAllStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>clearLog_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>840</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>initStrategy_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>CTAStrategyManager</receiver>
   <slot>StrategyTable_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addStrategy_clicked()</slot>
  <slot>initStrategy_clicked()</slot>
  <slot>startStrategy_clicked()</slot>
  <slot>stopStrategy_clicked()</slot>
  <slot>initAllStrategy_clicked()</slot>
  <slot>startAllStrategy_clicked()</slot>
  <slot>stopAllStrategy_clicked()</slot>
  <slot>clearLog_clicked()</slot>
  <slot>StrategyTable_clicked()</slot>
 </slots>
</ui>
